PSReadLine/Cmdlets.cs

```diff
@@ -63,6 +63,12 @@ public enum AddToHistoryOption
         MemoryAndFile
     }
 
+    public enum PredictionStyle
```

Comment:
>PredictionStyle [](start = 16, length = 15)

Consider `PredictionDisplayStyle` or `PredictionViewStyle` instead.

Reply:
Given the clarification above, I prefer to keep the name as is.

Reply:
Renamed to `PredictionViewStyle`